app/models/concerns/decidim/reporting_proposals/participatory_space_role_config/valuator_override.rb

```diff
@@ -7,8 +7,11 @@ module ValuatorOverride
         extend ActiveSupport::Concern
 
         included do
+          # store the current value class-wide so it is compatible with other module adding accepted_components elements
+          @@currently_accepted_components = new(nil).accepted_components
+
           def accepted_components
-            [:proposals, :reporting_proposals]
+            @accepted_components ||= @@currently_accepted_components + [:reporting_proposals]
```

Comment:
The proposed solution may work... I am thinking (not tested) that we may use a simple 
```
 def accepted_components
    super + [:reporting_proposals] 
end
```